pkg/arvo/lib/chat/eval.hoon

```diff
@@ -2,15 +2,21 @@
 ++  eval
   |=  [=bowl:gall =hoon]
   ^-  (list tank)
-  =/  subj=[our=@p now=@da eny=@uvJ]
+  =/  fowl=[our=@p now=@da eny=@uvJ]
     :+  our.bowl
       now.bowl
     (shaz (cat 3 (mix [now eny]:bowl) %eny))
   ::
-  ;;  (list tank)
-  =<  +>
-  %+  mong
-    :-  mute
-    |.([(sell (slap (slop !>(subj) !>(..zuse)) hoon))]~)
-  |=(^ ~)
+  =/  subject  [fowl ..zuse]
+  =/  minted=(each [=type =nock] (list tank))
+    %-  mule  |.
+    (~(mint ut -:!>(subject)) %noun hoon)
+  ?:  ?=(%| -.minted)  p.minted
+  =/  =toon
+    (mock [subject nock.p.minted] |=(^ ~))
+  ?-  -.toon
+    %0  [(sell type.p.minted p.toon) ~]
+    %1  [leaf+"scries unsupported in chat eval" ~]
```

Comment:
I'm on the fence wrt the word "scry", and entirely opposed to the plural form. Also, it'd be nice to print the path.

Something like:

```
:-  leaf+".^ is not supported in chat"
(turn ;;((list path) p.toon) smyt)
```